build.gradle

```diff
@@ -39,6 +40,13 @@ intellij {
     println "Building for IntelliJ version: ${version}"
 
     alternativeIdePath idePath
+
+    if ("DOKI" == System.getenv("DEV_ENV")) {
+        setPlugins(
```

Comment:
I got tired of removing this every time before submitting a PR.
So I just put the conditional statement which only executes under a specific run configuration.